docs/control-structures.rst

```diff
@@ -55,6 +58,8 @@ Internal functions (marked with the ``@internal`` decorator) are only accessible
     def calculate(amount: uint256) -> uint256:
         return self._times_two(amount)
 
+.. note::
+    It is possible to define an ``internal`` function that uses default parameters (e.g. ``_times_two(a: uint256=1)``) at function declaration. Since calling an ``internal`` function is realized by jumping to its entry label, the internal function dispatcher ensures the correctness of the jumps. Please note that for the usage of default parameters via ``internal`` functions, Vyper versions ``>=0.3.8`` are strongly recommended due to the security advisory `GHSA-ph9x-4vc9-m39g <https://github.com/vyperlang/vyper/security/advisories/GHSA-ph9x-4vc9-m39g>`_.
```

Comment:
we can put the example directly into the example code above.

also i think the advisory reference is appropriate in the note, we can be a bit more specific tho - `Please note that for the usage ``internal`` functions with more than one default parameter, Vyper versions ``>=0.3.8`` are strongly recommended ...`